src/Intuition.php

```diff
@@ -356,11 +356,14 @@ protected function normalizeDomain( string $domain ) : string {
 	}
 
 	/**
+	 * Normalize a language string (which may be user-supplied and unsafe) to be an all-lowercase
+	 * hyphen-separated string that approximately conforms to IETF's 'Tags for Identifying Languages'.
+	 * @link https://tools.ietf.org/html/rfc5646
 	 * @param string $lang
 	 * @return string
 	 */
 	protected function normalizeLang( $lang ) {
```

Comment:
This  means the function can now return empty string for a non-empty input, which breaks expectation of most internal uses of this function.

We may want to move this to setLang for now, so that we continue to require internal calls to be valid already (except for case sensitivity).

Reply:
Sure thing; I'll update it now.

What internal calls would be broken by this though?

Reply:
Off-hand, the main one that comes to mind is where the original string is checked for being empty, and based that we either normalise it, or use a default.

Reply:
Good point.

I've updated the patch; see what you think.